v1.2.2.3rc content - updated

bethdingley opened the pull request:
@JamesAnstey export is ready to have new branded names created! 

@sol1105 are you able to quickly check the consistency across the raw and release versions? 

@rigoudyg if there's any checks you need to do, feel free! 

I'm hoping we can release the content tomorrow or Friday if not.

bethdingley committed:
v1.2.2.3rc content

rigoudyg commented:
Thanks for preparing the version @bethdingley . I have spotted a first issue in both exports, there are two BCVs variables associated with  the following variables in Variables table :
- recKMHaEeWOGW8lIC atmos.ta.tavg-p19-hxy-air.day.glb
- recUvpxbQWeTeKKPC atmos.hus.tavg-p19-hxy-u.day.glb
- reciYyeFZgWguDPxG atmos.va.tavg-p19-hxy-air.day.glb
- recwcM4y2MfMp7xrP atmos.ua.tavg-p19-hxy-air.day.glb
Is it expected?

bethdingley commented:
@rigoudyg Yes that's expected!

rigoudyg commented:
Thanks @bethdingley , I will make a PR to solve this issue.
I have a difference between release and raw exports on variables list for two variables groups.
The variables are : 
- aerosol.emilnox.tavg-al-hxy-u.mon.glb
- atmosChem.emiavnox.tavg-al-hxy-u.mon.glb

In raw export, they are linked to group aerchemmip_3d_monthly.
In release one, to aerchemmip_2d_monthly

bethdingley commented:
Ah thanks, I will fix that!

sol1105 commented:
### CF Standard Names
 **Field 'Physical parameters'**
  - 80ac3216-a698-11ef-914a-613c0433d878
    - unique in raw (3): ['534001c2fd879bfda1d9b66d0a61144c', 'f86eb67d-8ca1-11ef-944e-41a8eb05f654', '691673a210102ac652eed2b784dd2ab4']
  - 3ba8de65-8ca2-11ef-944e-41a8eb05f654
    - unique in release (3): ['534001c2fd879bfda1d9b66d0a61144c', 'f86eb67d-8ca1-11ef-944e-41a8eb05f654', '691673a210102ac652eed2b784dd2ab4']

### Cell Methods
**Field 'title'**
   - a269a4d6-8c9b-11ef-944e-41a8eb05f654
     - release: 'Time Mean of Area Sum on Sea-ice'
     - raw:  'Time Mean of Area Sum on Sea'

### ESM BSVs
**Field 'Variables'**
There seem to be different Variables assigned to BSVs in the raw and release exports in most cases, example:
- '80ab7355-a698-11ef-914a-613c0433d878':
  - release:  'baa507f2-e5dd-11e5-8482-ac72891c3257'
  - raw: 'bab7c2d4-e5dd-11e5-8482-ac72891c3257'

**Field 'Physical Parameters'**
In two cases, no physical parameter is assigned for the raw export:
- '80ab7384-a698-11ef-914a-613c0433d878'
  - only release export has a physical parameter assigned: 'f86eb6b8-8ca1-11ef-944e-41a8eb05f654'
- '80ab7383-a698-11ef-914a-613c0433d878'
  - only release export has a physical parameter assigned: '7539774a693b6a99db7e174343a488bd'
  
### Opportunities
- Field `Count (Unique list of variables attached to Opportunity)` does not exist for the raw export
- Field `Unique list of variables attached to Opportunity` does not exist for the raw export

### Physical Parameters
**Field 'Units'**
- 534001c2fd879bfda1d9b66d0a61144c
  - release: kg m-2 s-1
  - raw: mol s-1
-  f86eb67d-8ca1-11ef-944e-41a8eb05f654
  - release:  kg m-2 s-1
  - raw: mol m-2 s-1 

**Field 'CF Standard Name'**
- 534001c2fd879bfda1d9b66d0a61144c
  - unique in raw (1): ['80ac3216-a698-11ef-914a-613c0433d878']
  - unique in release (1): ['3ba8de65-8ca2-11ef-944e-41a8eb05f654']
- f86eb67d-8ca1-11ef-944e-41a8eb05f654
  - unique in raw (1): ['80ac3216-a698-11ef-914a-613c0433d878']
  - unique in release (1): ['3ba8de65-8ca2-11ef-944e-41a8eb05f654']
- 691673a210102ac652eed2b784dd2ab4
  - unique in raw (1): ['80ac3216-a698-11ef-914a-613c0433d878']
  - unique in release (1): ['3ba8de65-8ca2-11ef-944e-41a8eb05f654']
 
**Field 'Processing Note'**
- '590ef524-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5914d6d8-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5913e674-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '59141748-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5914a456-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590e9390-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- 'f718ef1940e6feab018d81f508bd87c2'
  - release: 'UNDEFINED'
  - raw: ['si']
- '590f4f2e-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590e34fe-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '591523cc-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '59138238-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '80ab71e6-a698-11ef-914a-613c0433d878'
  - release: 'UNDEFINED'
  - raw: ['si']
- '590f0442-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- 'f86eb6ce-8ca1-11ef-944e-41a8eb05f654'
  - release: 'UNDEFINED'
  - raw: ['si']
- '590e9070-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590dfb2e-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '590e80c6-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5914c95e-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '2a92e434b37059d90e72b193bdbcce0f'
  - release: 'UNDEFINED'
  - raw: ['si', 'intensive']
- 'aa9073c2-1b36-11e6-a696-35cd2d8034df'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5914eb78-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590ed33c-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '80ab7255-a698-11ef-914a-613c0433d878'
  - release: 'UNDEFINED'
  - raw: ['si', 'ext_si']
- '590e80c6-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5914c95e-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '2a92e434b37059d90e72b193bdbcce0f'
  - release: 'UNDEFINED'
  - raw: ['si', 'intensive']
- 'aa9073c2-1b36-11e6-a696-35cd2d8034df'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5914eb78-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590ed33c-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '80ab7255-a698-11ef-914a-613c0433d878'
  - release:'UNDEFINED'
  - raw: ['si', 'ext_si']
- '5917ba9c-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5914e0ba-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590d70b4-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590f5672-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '591321f8-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '591733d8-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590e3ee0-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590ecbda-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '591774d8-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- 'f86eb6d2-8ca1-11ef-944e-41a8eb05f654'
  - release:'UNDEFINED'
  - raw: ['si']
- '590ea16e-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- 'f86eb6d0-8ca1-11ef-944e-41a8eb05f654'
  - release:'UNDEFINED'
  - raw: ['si']
- '590dfdc2-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- 'f86eb6cd-8ca1-11ef-944e-41a8eb05f654'
  - release:'UNDEFINED'
  - raw: ['si', 'intensive']
- '590d7924-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590df5e8-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5917369e-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- 'f86eb6d1-8ca1-11ef-944e-41a8eb05f654'
  - release:'UNDEFINED'
  - raw: ['si']
- '59142a3a-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '590dbb78-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590eb1ea-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '80ab71e7-a698-11ef-914a-613c0433d878'
  - release:'UNDEFINED'
  - raw: ['si']
- 'f86eb6cc-8ca1-11ef-944e-41a8eb05f654'
  - release:'UNDEFINED'
  - raw: ['si']
- '5917e51c-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- 'f86eb6cb-8ca1-11ef-944e-41a8eb05f654'
  - release:'UNDEFINED'
  - raw: ['si']
- '59131140-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590dac64-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '0353bea4-dca0-11e5-81c9-5404a60d96b5'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '59150da6-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590d7370-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '590dce42-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5913f77c-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '59139246-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '590d38b0-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '590e9656-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590d7654-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '5914a6b8-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '590e4bd8-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590d98b4-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590df8a4-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590f2f8a-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '59133288-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '53828236-bf01-11e6-a554-ac72891c3257'
  - release: 'UNDEFINED'
  - raw: ['si']
- '590de2ce-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '590d4fc6-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- 'f86eb6cf-8ca1-11ef-944e-41a8eb05f654'
  - release: 'UNDEFINED'
  - raw: ['si']
- '590e1ef6-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'ext_si']
- '5913c266-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']
- '590d2848-9e49-11e5-803c-0d0b866b59f3'
  - release: ['si']
  - raw: ['si', 'intensive']

### Variable Group
- "dafc744f-8c95-11ef-944e-41a8eb05f654" (paleo_cryosphere) lacks a lot of fields in the release export, also the UID field is not set, it has record_id: "recFQJunelDpDb4dc"

**Field 'Size (millions of data points)'**

- 'dafc73c9-8c95-11ef-944e-41a8eb05f654'
  - release: 46284.91200000006
  - raw: 47269.87200000007
-  'dafc73c5-8c95-11ef-944e-41a8eb05f654'
  - release: 1502.0640000000044
  - raw: 1485.6480000000042
 
**Field 'Number of variables in group'**
-  'dafc73c9-8c95-11ef-944e-41a8eb05f654'
  - release: 99
  - raw: 101
- 'dafc73c5-8c95-11ef-944e-41a8eb05f654'
  - release: 165
  - raw: 163

**Field 'Variables'**
- 'dafc73c9-8c95-11ef-944e-41a8eb05f654'
  - release: List of record ids with 99 elements
  - raw: List of record ids with 101 elements
    - unique UIDs in raw ['83bbfc0c-7f07-11ef-9308-b1dd71e64bec', '19bfbace-81b1-11e6-92de-ac72891c3257']
- 'dafc73c5-8c95-11ef-944e-41a8eb05f654'
  - release: List of record ids with 165 elements
    - unique UIDs in release ['83bbfc0c-7f07-11ef-9308-b1dd71e64bec', '19bfbace-81b1-11e6-92de-ac72891c3257']
  - raw: List of record ids with 163 elements

### Variables
- Fields `Flag meaning` and `Flag values` are missing in raw export

**Field 'CMIP7 Variable Groups'**
  - 7112fc9a-faa7-11e6-bfb7-ac72891c3257
    - unique in release (1): ['dafc744f-8c95-11ef-944e-41a8eb05f654']
  - 71220f64-faa7-11e6-bfb7-ac72891c3257
    - unique in release (1): ['dafc744f-8c95-11ef-944e-41a8eb05f654']
  - 710bfb0c-faa7-11e6-bfb7-ac72891c3257
    - unique in release (1): ['dafc744f-8c95-11ef-944e-41a8eb05f654']
  - 712a8130-faa7-11e6-bfb7-ac72891c3257
    - unique in release (1): ['dafc744f-8c95-11ef-944e-41a8eb05f654']
  - 83bbfc0c-7f07-11ef-9308-b1dd71e64bec
    - unique in raw (1): ['dafc73c9-8c95-11ef-944e-41a8eb05f654']
    - unique in release (1): ['dafc73c5-8c95-11ef-944e-41a8eb05f654']
  - 19bfbace-81b1-11e6-92de-ac72891c3257
    - unique in raw (1): ['dafc73c9-8c95-11ef-944e-41a8eb05f654']
    - unique in release (1): ['dafc73c5-8c95-11ef-944e-41a8eb05f654']
  - 714b545a-faa7-11e6-bfb7-ac72891c3257
    - unique in release (1): ['dafc744f-8c95-11ef-944e-41a8eb05f654']
  - 7147c57e-faa7-11e6-bfb7-ac72891c3257
    - unique in release (1): ['dafc744f-8c95-11ef-944e-41a8eb05f654']

**Field 'CF Standard Name (from Physical Parameter)'**
  - 19bfbace-81b1-11e6-92de-ac72891c3257
    - unique in raw (1): ['80ac3216-a698-11ef-914a-613c0433d878']
    - unique in release (1): ['3ba8de65-8ca2-11ef-944e-41a8eb05f654']
  - 83bbfc0c-7f07-11ef-9308-b1dd71e64bec
    - unique in raw (1): ['80ac3216-a698-11ef-914a-613c0433d878']
    - unique in release (1): ['3ba8de65-8ca2-11ef-944e-41a8eb05f654']
  - 19bf9c2e-81b1-11e6-92de-ac72891c3257
    - unique in raw (1): ['80ac3216-a698-11ef-914a-613c0433d878']
    - unique in release (1): ['3ba8de65-8ca2-11ef-944e-41a8eb05f654']

**Field 'ESM BSV 1.4'**
- same issues as above for the ESM BSV objects

**Field 'Units (from Physical Parameter)'**
- same issues as above for Physical Parameter objects

**Field Processing Notes**
There are quite a few issues like this:
- 'c9184416-c5e8-11e6-84e6-5404a60d96b5'
  - release:  'This is a regional subset of the variable. If you are producing the global equivalent of Omon.phycalcos, you should omit this regional South30 version.
 CHANGE SINCE CMIP6 in Dimensions - CMIP6:longitude latitude time CMIP7:longitude latitude time depth0m,
 CHANGE SINCE CMIP6 in Dimensions - CMIP6:longitude latitude time CMIP7:longitude latitude time osurf,'
   - raw:  'This is a regional subset of the variable. If you are producing the global equivalent of Omon.phycalcos, you should omit this regional South30 version.
 CHANGE SINCE CMIP6 in Dimensions - CMIP6:longitude latitude time CMIP7:longitude latitude time osurf,'

rigoudyg commented:
A few other differences between raw and release export:
- cell methods:
    - asms-tmn (uid a269a4d6-8c9b-11ef-944e-41a8eb05f654): title different (Sea or Sea-Ice respectively)
- opportunities:
    - image different for nearly all
    - additional "\n" in justification of resources
- physical parameters:
    - processing note different between the two exports
    - cf standard name and units different for:
        - emilnox (uid 534001c2fd879bfda1d9b66d0a61144c)
        - emianox (uid 691673a210102ac652eed2b784dd2ab4)
        - emiavnox (uid f86eb67d-8ca1-11ef-944e-41a8eb05f654)
- variables:
     - additional " " or "\n" in title for:
          - atmos.rsdsdiff.tavg-u-hxy-u.1hr.glb
          - ocean.hfds.tavg-u-hxy-sea.3hr.glb
          - ocean.rsdo.tavg-ol-hxy-sea.day.glb
          - ocean.rsntds.tavg-u-hxy-sea.3hr.glb
          - seaIce.rlus.tavg-u-hxy-si.day.glb
          - seaIce.rsds.tavg-u-hxy-si.day.glb
          - seaIce.siage.tavg-u-hxy-si.day.glb
          - seaIce.sifb.tavg-u-hxy-si.day.glb
          - seaIce.siflcondbot.tavg-u-hxy-si.day.glb
          - seaIce.siflcondtop.tavg-u-hxy-si.day.glb
          - seaIce.sifllattop.tavg-u-hxy-si.day.glb
          - seaIce.siflsensbot.tavg-u-hxy-si.day.glb
          - seaIce.siflsenstop.tavg-u-hxy-si.day.glb
          - seaIce.siitdsnthick.tavg-u-hxy-si.day.glb
          - seaIce.siitdthick.tavg-u-hxy-si.day.glb
          - seaIce.simpconc.tavg-u-hxy-si.day.glb
          - seaIce.simprefrozen.tavg-u-hxy-simp.day.glb
          - seaIce.simpthick.tavg-u-hxy-si.day.glb
          - seaIce.sirdgconc.tavg-u-hxy-si.day.glb
          - seaIce.sisali.tavg-u-hxy-si.day.glb
          - seaIce.sisaltmass.tavg-u-hxy-si.day.glb
          - seaIce.sitempbot.tavg-u-hxy-si.day.glb
          - seaIce.sitempsnic.tavg-u-hxy-si.day.glb
          - seaIce.sithick.tavg-u-hxy-sir.day.glb
     - missing flag meanings and flag values in raw for:
          - ocean.basin.ti-u-hxy-u.fx.glb
     - processing notes different for:
          - ocnBgchem.arag.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.bacc.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.calc.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.chl.tavg-d0m-hxy-sea.day.glb
          - ocnBgchem.chl.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.chlcalc.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.chldiat.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.chldiaz.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.chlmisc.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.chlpico.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.dfe.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.dissi13c.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.dissi14cabio.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.dissic.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.dmso.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.nh4.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.no3.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.o2.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.ph.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.phyc.tavg-d0m-hxy-sea.day.glb
          - ocnBgchem.phyc.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.phycalc.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.phydiat.tavg-d0m-hxy-sea.day.glb
          - ocnBgchem.phydiat.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.phydiaz.tavg-d0m-hxy-sea.day.glb
          - ocnBgchem.phydiaz.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.phymisc.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.phypico.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.si.tavg-d0m-hxy-sea.mon.glb
          - ocnBgchem.zmeso.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.zmicro.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.zmisc.tavg-d0m-hxy-sea.mon.30S-90S
          - ocnBgchem.zooc.tavg-d0m-hxy-sea.mon.30S-90S
     - esm-bcv different for (uid differs):
         -  atmos.areacella.ti-u-hxy-u.fx.glb
         - atmos.cl.tavg-al-hxy-u.mon.glb
         - atmos.cli.tavg-al-hxy-u.mon.glb
         - atmos.clivi.tavg-u-hxy-u.mon.glb
         - atmos.clt.tavg-u-hxy-u.day.glb
         - atmos.clt.tavg-u-hxy-u.mon.glb
         - atmos.clw.tavg-al-hxy-u.mon.glb
         - atmos.clwvi.tavg-u-hxy-u.mon.glb
         - atmos.evspsbl.tavg-u-hxy-u.mon.glb
         - atmos.hfls.tavg-u-hxy-u.mon.glb
         - atmos.hfss.tavg-u-hxy-u.mon.glb
         - atmos.hur.tavg-p19-hxy-air.mon.glb
         - atmos.hur.tavg-p19-hxy-u.day.glb
         - atmos.hurs.tavg-h2m-hxy-u.6hr.glb
         - atmos.hurs.tavg-h2m-hxy-u.day.glb
         - atmos.hurs.tavg-h2m-hxy-u.mon.glb
         - atmos.hus.tavg-p19-hxy-u.day.glb
         - atmos.hus.tavg-p19-hxy-u.mon.glb
         - atmos.huss.tavg-h2m-hxy-u.day.glb
         - atmos.huss.tavg-h2m-hxy-u.mon.glb
         - atmos.huss.tpt-h2m-hxy-u.3hr.glb
         - atmos.pr.tavg-u-hxy-u.1hr.glb
         - atmos.pr.tavg-u-hxy-u.3hr.glb
         - atmos.pr.tavg-u-hxy-u.day.glb
         - atmos.pr.tavg-u-hxy-u.mon.glb
         - atmos.prc.tavg-u-hxy-u.mon.glb
         - atmos.prsn.tavg-u-hxy-u.mon.glb
         - atmos.prw.tavg-u-hxy-u.mon.glb
         - atmos.ps.tavg-u-hxy-u.day.glb
         - atmos.ps.tavg-u-hxy-u.mon.glb
         - atmos.psl.tavg-u-hxy-u.day.glb
         - atmos.psl.tavg-u-hxy-u.mon.glb
         - atmos.rlds.tavg-u-hxy-u.mon.glb
         - atmos.rldscs.tavg-u-hxy-u.mon.glb
         - atmos.rlus.tavg-u-hxy-u.mon.glb
         - atmos.rluscs.tavg-u-hxy-u.mon.glb
         - atmos.rlut.tavg-u-hxy-u.mon.glb
         - atmos.rlutcs.tavg-u-hxy-u.mon.glb
         - atmos.rsds.tavg-u-hxy-u.day.glb
         - atmos.rsds.tavg-u-hxy-u.mon.glb
         - atmos.rsdscs.tavg-u-hxy-u.mon.glb
         - atmos.rsdt.tavg-u-hxy-u.mon.glb
         - atmos.rsus.tavg-u-hxy-u.mon.glb
         - atmos.rsuscs.tavg-u-hxy-u.mon.glb
         - atmos.rsut.tavg-u-hxy-u.mon.glb
         - atmos.rsutcs.tavg-u-hxy-u.mon.glb
         - atmos.sfcWind.tavg-h10m-hxy-u.day.glb
         - atmos.sfcWind.tavg-h10m-hxy-u.mon.glb
         - atmos.sftlf.ti-u-hxy-u.fx.glb
         - atmos.ta.tavg-p19-hxy-air.day.glb
         - atmos.ta.tavg-p19-hxy-air.mon.glb
         - atmos.ta.tpt-p3-hxy-air.6hr.glb
         - atmos.tas.tavg-h2m-hxy-u.day.glb
         - atmos.tas.tavg-h2m-hxy-u.mon.glb
         - atmos.tas.tmax-h2m-hxy-u.day.glb
         - atmos.tas.tmaxavg-h2m-hxy-u.mon.glb
         - atmos.tas.tmin-h2m-hxy-u.day.glb
         - atmos.tas.tminavg-h2m-hxy-u.mon.glb
         - atmos.tas.tpt-h2m-hxy-u.3hr.glb
         - atmos.tauu.tavg-u-hxy-u.mon.glb
         - atmos.tauv.tavg-u-hxy-u.mon.glb
         - atmos.ts.tavg-u-hxy-u.mon.glb
         - atmos.ua.tavg-p19-hxy-air.day.glb
         - atmos.ua.tavg-p19-hxy-air.mon.glb
         - atmos.ua.tpt-p3-hxy-air.6hr.glb
         - atmos.uas.tavg-h10m-hxy-u.day.glb
         - atmos.uas.tavg-h10m-hxy-u.mon.glb
         - atmos.va.tavg-p19-hxy-air.day.glb
         - atmos.va.tavg-p19-hxy-air.mon.glb
         - atmos.va.tpt-p3-hxy-air.6hr.glb
         - atmos.vas.tavg-h10m-hxy-u.day.glb
         - atmos.vas.tavg-h10m-hxy-u.mon.glb
         - atmos.wap.tavg-p19-hxy-air.mon.glb
         - atmos.wap.tavg-p19-hxy-u.day.glb
         - atmos.zg.tavg-p19-hxy-air.day.glb
         - atmos.zg.tavg-p19-hxy-air.mon.glb
         - land.evspsblsoi.tavg-u-hxy-lnd.mon.glb
         - land.evspsblveg.tavg-u-hxy-lnd.mon.glb
         - land.lai.tavg-u-hxy-lnd.mon.glb
         - land.mrro.tavg-u-hxy-lnd.mon.glb
         - land.mrros.tavg-u-hxy-lnd.mon.glb
         - land.mrso.tavg-u-hxy-lnd.mon.glb
         - land.mrsofc.ti-u-hxy-lnd.fx.glb
         - land.mrsol.tavg-d10cm-hxy-lnd.mon.glb
         - land.orog.ti-u-hxy-u.fx.glb
         - land.rootd.ti-u-hxy-lnd.fx.glb
         - land.sftgif.ti-u-hxy-u.fx.glb
         - land.slthick.ti-sl-hxy-lnd.fx.glb
         - landIce.mrfso.tavg-u-hxy-lnd.mon.glb
         - landIce.snc.tavg-u-hxy-lnd.mon.glb
         - landIce.snw.tavg-u-hxy-lnd.mon.glb
         - ocean.areacello.ti-u-hxy-u.fx.glb
         - ocean.basin.ti-u-hxy-u.fx.glb
         - ocean.bigthetao.tavg-ol-hxy-sea.mon.glb
         - ocean.deptho.ti-u-hxy-sea.fx.glb
         - ocean.hfds.tavg-u-hxy-sea.mon.glb
         - ocean.hfgeou.ti-u-hxy-sea.fx.glb
         - ocean.masscello.tavg-ol-hxy-sea.mon.glb
         - ocean.masscello.ti-ol-hxy-sea.fx.glb
         - ocean.mlotst.tavg-u-hxy-sea.mon.glb
         - ocean.sftof.ti-u-hxy-u.fx.glb
         - ocean.so.tavg-ol-hxy-sea.mon.glb
         - ocean.sos.tavg-u-hxy-sea.day.glb
         - ocean.sos.tavg-u-hxy-sea.mon.glb
         - ocean.tauuo.tavg-u-hxy-sea.mon.glb
         - ocean.tauvo.tavg-u-hxy-sea.mon.glb
         - ocean.thkcello.ti-ol-hxy-sea.fx.glb
         - ocean.tos.tavg-u-hxy-sea.day.glb
         - ocean.tos.tavg-u-hxy-sea.mon.glb
         - ocean.umo.tavg-ol-hxy-sea.mon.glb
         - ocean.uo.tavg-ol-hxy-sea.mon.glb
         - ocean.vmo.tavg-ol-hxy-sea.mon.glb
         - ocean.vo.tavg-ol-hxy-sea.mon.glb
         - ocean.wmo.tavg-ol-hxy-sea.mon.glb
         - ocean.wo.tavg-ol-hxy-sea.mon.glb
         - ocean.zos.tavg-u-hxy-sea.day.glb
         - ocean.zos.tavg-u-hxy-sea.mon.glb
         - ocean.zostoga.tavg-u-hm-sea.mon.glb
         - seaIce.siconc.tavg-u-hxy-u.day.glb
         - seaIce.siconc.tavg-u-hxy-u.mon.glb
         - seaIce.simass.tavg-u-hxy-si.mon.glb
         - seaIce.sithick.tavg-u-hxy-si.mon.glb
         - seaIce.sitimefrac.tavg-u-hxy-sea.mon.glb
         - seaIce.siu.tavg-u-hxy-si.mon.glb
         - seaIce.siv.tavg-u-hxy-si.mon.glb
         - seaIce.snd.tavg-u-hxy-sn.mon.glb
         - seaIce.ts.tavg-u-hxy-si.mon.glb

bethdingley commented:
Thanks @sol1105! All fixed below, except the missing field `Unique list of variables attached to Opportunity` in the raw - that looks like it should be there in the Airtable backend...

> ### CF Standard Names
>
> ### Cell Methods
> 
> ### ESM BSVs

All above fixed.

> ### Opportunities
> * Field `Count (Unique list of variables attached to Opportunity)` does not exist for the raw export

Fixed
> * Field `Unique list of variables attached to Opportunity` does not exist for the raw export

This should exist in raw export with the name 'Unique list of variables attached to Opportunity (linked)' - I don't believe this has changed since the last release either.
> 
> ### Physical Parameters

All fixed
> 
> ### Variable Group

All fixed

> ### Variables
> * Fields `Flag meaning` and `Flag values` are missing in raw export

Fixed

> **Field 'CMIP7 Variable Groups'**

All fixed!

bethdingley commented:
> A few other differences between raw and release export:
> 
> * cell methods:
>   
> * opportunities:
>   
> * physical parameters:
>   
> * variables:
>   
>   * additional " " or "\n" in title for:
>     
>   * missing flag meanings and flag values in raw for:
>     
>   * esm-bcv different for (uid differs):
>     

Thanks @rigoudyg! I think I've fixed what I can - erroneous whitespaces and "\n" are an annoying Airtable feature.

bethdingley commented:
@JamesAnstey - none of the changes above should affect the BCV name generation so you can still go ahead with that from the release export and the fixes to the above will appear when I re-export.

JamesAnstey commented:
> @JamesAnstey - none of the changes above should affect the BCV name generation so you can still go ahead with that from the release export and the fixes to the above will appear when I re-export.

Thanks @bethdingley - here are json & csv files (same content in each) with the regenerated branded names:
https://hpfx.collab.science.gc.ca/~rja001/CMIP7_data_request/v1.2.2.3_branded_names/
The version of the branded mapper I used is 0.13.0 (latest) with the simple addition of verticalLabelDD `ols` - PR for this is here:
https://github.com/znicholls/CMIP-branded-variable-mapper/pull/37

JamesAnstey commented:
Here's an updated draft of v1.2.2.3 release notes (a rough initial draft [was here](https://github.com/CMIP-Data-Request/CMIP7_DReq_Software/issues/198) - closing that issue since the Content repo is more appropriate for content release notes). Others please update/comment if these notes miss / misunderstand anything. 

The level of detail is intentionally high so that data producers already using the DR can easily see what has changed. I thought it useful to categorize the updates into subsections to make it more digestible, but happy for other suggestions of how to organize them (or better subsection titles...).

------------------

### Updates to experiments

- ScenarioMIP experiment names are finalized. The updated names replace the temporary placeholder names that were used in v1.2.2.2. 

- TIPMIP experiment names have been changed to provisional ones, as they currently are not compatible with CMIP7 CV requirements (they are too long)

### Updates to information about the Data Request

- Increased coverage by MIP tagging of Opportunities and Variable Groups

- The [DR web viewer](https://cmip-data-request.github.io/cmip7-dreq-webview/index.html) has a new tool for displaying changes in variables between DR versions: https://cmip-data-request.github.io/cmip7-dreq-webview/variable_changes.html

- Airtable Variables table: explanatory text added for the "List of Experiments" column indicating that listed experiments are those for which the DR variable has been requested by *any* Opportunity. This means that filtering the Variables table by Opportunity will not filter the experiments listed in this column. 

### Updates to requested variables

- Ocean surface variables for which the CF standard name doesn't specify ocean surface have been changed from `depth0m` coordinate to new `osurf` coordinate (46 DR variables).

- Monthly AerChemMIP variables `emianox`, `emiavnox`, `emilnox` (CMIP6 table name: `AERmon`) are layer-integrated with CF standard name `tendency_of_atmosphere_mass_content_of_nox_expressed_as_nitrogen_due_to_emission`.

- `height2m` coordinate is added to `LImon.tasIs`, `E1hr.tas`, `E1hr.tasSouth30`

- `sdepth10cm` added to `3hr.tsl`

### Updates to descriptive metadata

- long names (Title in Airtable) of some DR variables have been updated so that they have the same long name as other DR variables that share the same branded variable. 16 DR variables do not have the same long name as their branded variables, which are sea ice variables for which the DR variable long name indicates the region ("North" for NH, "South" for SH) (branded variables: `sivol_tavg-u-hm-u`, `sisnmass_tavg-u-hm-si`, `siextent_tavg-u-hm-u`, `siarea_tavg-u-hm-u`). 

- Region names are now in lower case instead of upper case (e.g. `glb` instead of `GLB`) with the exception of `30S-90S` which is unchanged. This affects the CMIP7 compound names of DR variables since these compound names include the region (e.g., `Amon.tas` is `atmos.tas.tavg-h2m-hxy-u.mon.glb`). 

- AerChemMip variables on `height2m` have had long names and comments (in the Airtable these are Title & Description respectively) corrected to reflect the use of 2m height coordinate. This replaces references to lowest model layer, `lowerModelLayer`, on which these were defined prior to v1.2.2.2.

- `flag_meanings` and `flag_values` attributes added for `Ofx.basin` / `ocean.basin.ti-u-hxy-u.fx.glb`, so that this information is easily machine-readable (the values/meanings are the same as CMIP6, but previously in the CMIP7 DR were only indicated in the Processing Note).

- The cell measures values `::MODEL` and `::OPT` are changed back to `--MODEL` and `--OPT`, which are understood by CMOR (the `::` versions will produce a CMOR error).

bethdingley committed:
v1.2.2.3rc - updated branded names

bethdingley commented:
@rigoudyg, @JamesAnstey and @sol1105 updated content! Hopefully issues are all fixed. 

Hopefully we can merge in and do release with this, just waiting for Karl to confirm whether we need to do a final update to branded names...

rigoudyg commented:
It works for me (provided integration of PR https://github.com/CMIP-Data-Request/CMIP7_DReq_Software/pull/199
Thanks @bethdingley!

rigoudyg commented:
@bethdingley, just one thing: there is no CF standard name for release export of eminox

bethdingley commented:
Thanks, I'll fix that and re-export!

bethdingley committed:
v1.2.2.3rc - fixing eminox

bethdingley commented:
Should be fixed now (it looked like the CF standard name was missing in the raw rather than the release)

bethdingley commented:
Updated release notes:

------------------

### Updates to experiments

- ScenarioMIP experiment names are finalized. The updated names replace the temporary placeholder names that were used in v1.2.2.2. 

- TIPMIP experiment names have been changed to provisional ones, as they currently are not compatible with CMIP7 CV requirements (they are too long)

### Updates to information about the Data Request

- Increased coverage by MIP tagging of Opportunities and Variable Groups

- The [DR web viewer](https://cmip-data-request.github.io/cmip7-dreq-webview/index.html) has a new tool for displaying changes in variables between DR versions: https://cmip-data-request.github.io/cmip7-dreq-webview/variable_changes.html

- Airtable Variables table: explanatory text added for the "List of Experiments" column indicating that listed experiments are those for which the DR variable has been requested by *any* Opportunity. This means that filtering the Variables table by Opportunity will not filter the experiments listed in this column. 

### Updates to requested variables

- Ocean surface variables for which the CF standard name doesn't specify ocean surface have been changed from `depth0m` coordinate to new `osurf` coordinate (46 DR variables).

- Monthly AerChemMIP variables `emianox`, `emiavnox`, `emilnox` (CMIP6 table name: `AERmon`) are layer-integrated with CF standard name `tendency_of_atmosphere_mass_content_of_nox_expressed_as_nitrogen_due_to_emission`.

- `height2m` coordinate is added to `LImon.tasIs`, `E1hr.tas`, `E1hr.tasSouth30`

### Updates to descriptive metadata

- long names (Title in Airtable) of some DR variables have been updated so that they have the same long name as other DR variables that share the same branded variable. 16 DR variables do not have the same long name as their branded variables, which are sea ice variables for which the DR variable long name indicates the region ("North" for NH, "South" for SH) (branded variables: `sivol_tavg-u-hm-u`, `sisnmass_tavg-u-hm-si`, `siextent_tavg-u-hm-u`, `siarea_tavg-u-hm-u`). 

- Region names are now in lower case instead of upper case (e.g. `glb` instead of `GLB`) with the exception of `30S-90S` which is unchanged. This affects the CMIP7 compound names of DR variables since these compound names include the region (e.g., `Amon.tas` is `atmos.tas.tavg-h2m-hxy-u.mon.glb`). 

- AerChemMip variables on `height2m` have had long names and comments (in the Airtable these are Title & Description respectively) corrected to reflect the use of 2m height coordinate. This replaces references to lowest model layer, `lowerModelLayer`, on which these were defined prior to v1.2.2.2.

- `flag_meanings` and `flag_values` attributes added for `Ofx.basin` / `ocean.basin.ti-u-hxy-u.fx.glb`, so that this information is easily machine-readable (the values/meanings are the same as CMIP6, but previously in the CMIP7 DR were only indicated in the Processing Note).

- The cell measures values `::MODEL` and `::OPT` are changed back to `--MODEL` and `--OPT`, which are understood by CMOR (the `::` versions will produce a CMOR error).
